registry.k8s.io/images/k8s-staging-cluster-api/images.yaml

```diff
@@ -253,6 +254,7 @@
     "sha256:f588fbe6f73160981cb7ae8ed9d1e48ea0322ddf3ec28125532cb0af01d91ce5": ["v1.10.5"]
     "sha256:913e0938a27a02947a24becc6b409511d5bfb4f95c6d51eb4137243c93f00b11": ["v1.10.6"]
     "sha256:a8f8975d97a7e1feaaf11368a92beb30dcbeedf1df844267840714f522660ad7": ["v1.10.7"]
+    "sha256:54cb066f99b032599de515975ae16288c3bc27693068d632a051cb3660ce323c": ["v1.10.8"]
```

Comment:
SHA `54cb066f99b0` matches clusterctl v1.10.8 at the [staging repo](https://console.cloud.google.com/gcr/images/k8s-staging-cluster-api/global/clusterctl@sha256:54cb066f99b032599de515975ae16288c3bc27693068d632a051cb3660ce323c/details).